HelperFunctions.ps1

```diff
@@ -1299,3 +1299,18 @@ function ReplaceCDN {
     }
     $sourceUrl
 }
+
+function Write-GroupStart([string] $Message) {
+    switch ($true) {
+        $bcContainerHelperConfig.IsAzureDevOps { Write-Host "##[group]$Message"; break }
+        $bcContainerHelperConfig.IsGitHubActions { Write-Host "::group::$Message"; break }
```

Comment:
Please change the order of these to have IsGitHub on top and test for IsAzureDevOps afterwards.

I can see situations where TF_BUILD could be set to something in GitHub - something a partner could define themselves. I don't see anybody defining an environment variable called GITHUB_ACTIONS themselves.

Reply:
fixed